### PR TITLE
Add new parameters for scripting usage

### PR DESCRIPTION
**Pull request by whizz**
Added two new parameters which are suitable if you have many addresses to process and you want to script the thing. See updated README for syntax.

I cleaned up the previous merge request from whitespace changes.

### CODE REVIEW

**Comment by avialias** [comment]
Can you eli5 me how to use multi-address mode and your parametars?
Need to automatize claiming process.
I have about 30 priv keys to sweep on every fork and that takes time. 

How to format priv keys in txt file, how to write py command and how to start it ?

Thx.

**Comment by whizz** [comment]
It's probably not "eli5" level, but I wrote about it here: https://medium.com/@mtaborsky/dumping-bitcoin-forks-bitcoinx-ea6d88e0ae68. Basically:

1.  I extract private keys and addresses into CSV from a seed (described here https://medium.com/@mtaborsky/dumping-bitcoin-forks-bitcore-2b6501bf33f3). If you already have something like that, just put it in the same format.
2. Run it thru a simple PHP script to generate the correct commands for claimer script (whole script is included in https://medium.com/@mtaborsky/dumping-bitcoin-forks-bitcoinx-ea6d88e0ae68)
3. Save it into a file `myscript.sh`
4. Run `bash myscript.sh`
5. Profit!

**Comment by avialias** [comment]
Well I just investigate you a little.
Sailing CRO - Montenegro.

Croatia here... :)
